openqabot/types/incidents.py

```diff
@@ -96,35 +96,37 @@ def __call__(
                             inc.id,
                         )
                         continue
+
+                    if inc.staging:
+                        continue
+
+                    flavor_settings = self.settings.copy()
+                    if "flavor_settings" in data:
+                        flavor_settings.update(data["flavor_settings"])
+
+                    if "packages" in data:
+                        if not inc.contains_package(data["packages"]):
+                            continue
+
+                    if "excluded_packages" in data:
+                        if inc.contains_package(data["excluded_packages"]):
+                            continue
+
                     full_post: Dict[str, Any] = {}
                     full_post["api"] = "api/incident_settings"
                     full_post["qem"] = {}
                     full_post["openqa"] = {}
-                    full_post["openqa"].update(self.settings)
                     full_post["qem"]["incident"] = inc.id
                     full_post["openqa"]["ARCH"] = arch
                     full_post["qem"]["arch"] = arch
                     full_post["openqa"]["FLAVOR"] = flavor
                     full_post["qem"]["flavor"] = flavor
-                    full_post["openqa"]["VERSION"] = self.settings["VERSION"]
```

Comment:
disagree , flavor specific settings should NOT allow to overwrite VERSION defined on product level , this will open gate for so many tricky bugs and data inconsistencies that we do NOT want to deal with